lang/smaily-et_EE.po

```diff
@@ -316,35 +316,47 @@ msgid "Additional Information"
 msgstr "Lisainformatsioon"
 
 #. Base ID.
-#: inc/Widget/SmailyWidget.php:21 inc/Widget/SmailyWidget.php:143
+#: inc/Widget/SmailyWidget.php:21 inc/Widget/SmailyWidget.php:158
 msgid "Smaily Newsletter"
 msgstr "Smaily Uudiskiri"
 
 #: inc/Widget/SmailyWidget.php:22
 msgid "Smaily for WooCommerce Newsletter Widget"
 msgstr "Smaily uudiskirjaga liitumise moodul"
 
-#: inc/Widget/SmailyWidget.php:111
+#: inc/Widget/SmailyWidget.php:97
+msgid "Subscription registration successful."
+msgstr "Registreerumine õnnestus."
+
+#: inc/Widget/SmailyWidget.php:100
+msgid "Subscription do not contain a valid email address."
+msgstr "Registreeritud email on vigane."
+
+#: inc/Widget/SmailyWidget.php:103
+msgid "Subscription registration failed."
+msgstr "Registreerimune ebaõnnestus."
```

Comment:
Typo here - is `Registreerimune` should be `Registreerumine`.

Reply:
Bollocks! https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/43/commits/908114bfdd9b5c85ae83fb794fd137fc13ab4870